sched/worker/fake/fake.go

```diff
@@ -13,35 +14,22 @@ func (c *noopWorkerController) RunAndWait(task sched.TaskDefinition) error {
 	return nil
 }
 
-func NewNoopWorker() WorkerController {
+func NewNoopWorker() worker.WorkerController {
 	return &noopWorkerController{}
 }
 
-type noopWorkerWrapper struct{}
-
-func (w *noopWorkerWrapper) Wrap(node cluster.Node) cluster.Node {
-	return &workerNode{
-		worker: &noopWorkerController{},
-		node:   node,
-	}
-}
-
-var NoopWorkerWrapper = &noopWorkerWrapper{}
-
-type waitingWorkerWrapper struct {
+func NoopWorkerFactory(node cluster.Node) worker.WorkerController {
```

Comment:
OK, I updated the function names, but I think we still need the type name to be factory (in worker.go). WDYT? Does that sound right?